hack/util.sh

```diff
@@ -5,6 +5,88 @@
 TIME_SEC=1000
 TIME_MIN=$((60 * $TIME_SEC))
 
+# configure_and_start_os will create and write OS master certificates, node config,
+# OS config and finally starts the OS server. Returns the PID of the OS server.
+# 
+# $1 - log dir
+function configure_and_start_os {
+  openshift admin ca create-master-certs \
+    --overwrite=false \
+    --cert-dir="${MASTER_CONFIG_DIR}" \
```

Comment:
I'm not great with bash, but is it possible to set variables like `MASTER_CONFIG_DIR` in a function and have the value persist all the way back to the caller?  It would be really nice if we could simply call this function, have it fill defaults if I hadn't already set anything and then allow the caller to the created values so it could derive things like CA location.